CHANGELOG.md

```diff
@@ -3,6 +3,87 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.15.0
```

Comment:
The minor is because of the `getRowsToExport` option?

Reply:
I'm open to suggestions. We can say it's a fix 😅

Reply:
It was just to be sure I did not missed another reason

Reply:
that and we also just added columns auto-sizing